### ISSUE TITLE
Windows 11: PreventStandby doesn't prevent standby any more

### ISSUE THREAD

**Issue by michaelherger** [LMS-Community/slimserver#746]
See https://forums.slimdevices.com/showthread.php?115704-Anyone-running-LMS-on-Windows-11&p=1042620&viewfull=1#post1042620

```
[22-01-07 12:16:18.0093] Slim::Plugin::PreventStandby::Plugin::_playersBusy (159) Player Squeezebox is busy...
[22-01-07 12:16:18.0095] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (109) Resetting idle counter. 20 minutes left in allowed idle period.
[22-01-07 12:16:18.0097] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (126) Preventing System Standby...
[22-01-07 12:17:18.0075] Slim::Plugin::PreventStandby::Plugin::_playersBusy (159) Player Squeezebox is busy...
[22-01-07 12:17:18.0078] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (109) Resetting idle counter. 20 minutes left in allowed idle period.
[22-01-07 12:17:18.0079] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (126) Preventing System Standby...
[22-01-07 12:18:18.0274] Slim::Plugin::PreventStandby::Plugin::_playersBusy (159) Player Squeezebox is busy...
[22-01-07 12:18:18.0275] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (109) Resetting idle counter. 20 minutes left in allowed idle period.
[22-01-07 12:18:18.0276] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (126) Preventing System Standby...
[22-01-07 12:19:18.0146] Slim::Plugin::PreventStandby::Plugin::_playersBusy (159) Player Squeezebox is busy...
[22-01-07 12:19:18.0149] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (109) Resetting idle counter. 20 minutes left in allowed idle period.
[22-01-07 12:19:18.0150] Slim::Plugin::PreventStandby::Plugin::checkClientA ctivity (126) Preventing System Standby...
```

at which point the PC goes to sleep.

**Comment by michaelherger** [comment]
https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-setthreadexecutionstate - should still work as before?

**Comment by mherger** [comment]
This is fixed in 8.3
